tests/python/unittest/test_tvmscript_bufferslice_loop_range.py

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I suggest moving this test to the round-trip test file? That makes sense to me and I think there's no need for a new file.

Reply:
DONE